gef.py

```diff
@@ -637,6 +637,7 @@ def __init__(self, addr):
         try:
             self.__addr = to_unsigned_long(gdb.parse_and_eval("&{}".format(addr)))
         except gdb.error:
+            warn("Could not parse address '&{}' when searching malloc_state struct, using '&main_arena' instead".format(addr))
```

Comment:
Nit: Line length. Break it up.
```suggestion
            warn("Could not parse address '&{}' when searching malloc_state struct, "
                 "using '&main_arena' instead".format(addr))
```